packages/components/src/toolbar/stories/index.story.tsx

```diff
@@ -181,3 +185,42 @@ WithoutGroup.args = {
 		</>
 	),
 };
+
+/**
+ * Set the variant to `unstyled` to remove default border styles.
+ * Otherwise, leave it as `undefined` for default styles.
+ */
+
+export const WithoutStyles = Template.bind( {} );
```

Comment:
I mentioned consistency with Popover...

So if we decide to keep the name 'unstyled', I'll update the story name to match.

Reply:
We could leave the story out. I added it because it's a visual change, so I thought seeing an explicit demo could be helpful. Also, `Popover` has the same story so I can add another point to my consistency list 😄 

But I'm happy to remove it if you don't think it adds enough value.

Reply:
Let's keep it for now, but maybe rewrite the `args` to reuse `Default.args` like suggested above>

Reply:
Sounds good!